tests/http/clients/async_flask.py

```diff
@@ -16,12 +16,12 @@
 from .flask import FlaskHttpClient
 
 
-class GraphQLView(BaseAsyncGraphQLView):
+class GraphQLView(BaseAsyncGraphQLView[dict[str, object], object]):
```

Comment:
we are reusing `[dict[str, object], object]` quite often for `Context, RootValue` in the specific framework views. Do you think it makes sense to factor this out into a separate intermediate layer?

Reply:
Fair enough! It would be nice to have a base class for test views that implements `get_context` and `get_root_value`. That would work decently well for `get_root_value`, every view that uses context vars (i.e., fastapi and litestar) would still override `get_context`. But yup, that could be done. I'll try and see whether it makes things better :)

Reply:
We could also do that in a separate pr to keep things here typing-only

Reply:
Good point, I'll do that instead.